Cognitive-Library/SentimentAnalyzer/Sentiments.cs

```diff
@@ -9,26 +9,47 @@
 
 namespace SentimentAnalyzer
 {
-    public class Sentiments
+    /// <summary>
+    /// Thread-safe singleton to manage initialization of ML training model
+    /// to improve runtime performance.
+    /// </summary>
+    public sealed class Sentiments
     {
-        public static SentimentPrediction Predict(string text)
+        private static readonly Sentiments instance = new Sentiments();
```

Comment:
This needs to be `[ThreadStatic]` and initialized once per thread

Reply:
Thanks, addressed in https://github.com/arafattehsin/CognitiveRocket/pull/5